micronaut-hello-rest-maven/README-Cloud-Shell.md

```diff
@@ -0,0 +1,224 @@
+# Micronaut Hello World REST App with GraalVM Enterprise in OCI Cloud Shell
+
+This sample shows how you can get started quickly with GraalVM Enterprise Edition in Oracle Cloud Infrastructre (OCI) Cloud Shell. This sample uses a simple hello world REST application built with the Micronaut framework and GraalVM Enterprise Native Image and JDK (Java Development Kit).
```

Comment:
I think I prefer "example", but that's a personal preference. It's probably more commonly used than  "sample".

Reply:
Changed to `example` in all the readme files.